zerver/lib/onboarding.py

```diff
@@ -314,6 +314,18 @@ def send_initial_realm_messages(realm: Realm) -> None:
             start_topic_help_url="/help/starting-a-new-topic",
         )
 
+        content_of_zulip_update_announcements_topic_name = (
+            _("This topic will receive automated updates about new Zulip features.") + "\n"
+            "\n"
+            + _(
+                "If you like, you can [configure](/#organization/organization-settings) a different stream "
+                "for these updates (and [move]({move_content_help_url}) the initial updates there), or "
+                "[turn this feature off](/#organization/organization-settings) altogether."
+            )
+        ).format(
+            move_content_help_url="/help/move-content-to-another-stream",
```

Comment:
Let's use variables, `settings_url` and `settings_disable_url` here, so we don't have to ask translators to change anything if the URLs change. (Right now, those URLs will be the same, but we might in the future be able to add a vertical-position-within-page anchor at some point)

Reply:
Makes sense. Made the changes.